src/main/kotlin/com/coxautodev/graphql/tools/Utils.kt

```diff
@@ -33,3 +34,21 @@ internal fun JavaType.unwrap(): Class<out Any> =
         } else {
             this as Class<*>
         }
+
+/**
+ * Simple heuristic to check is a method is a trivial data fetcher.
+ *
+ * Requirements are:
+ * prefixed with get
+ * must have zero parameters
```

Comment:
This should become more strict and only return true for 'primitives'.

E.g. any actual primitives and String, Integer, Double etc and the java date/time objects.

or If the field returns a scalar. that should work too